res/controllers/Traktor Kontrol S4 MK3.hid.xml

```diff
@@ -199,6 +214,33 @@
                     When shifting decks, the mixer controls microphones or auxiliary lines. If both a microphone and an auxiliary are configured on the same channel, the mixer will prioritize controlling the auxiliary.
                 </description>
             </option>
+            <option
+                variable="softwareMixerMain"
+                type="boolean"
+                default="false"
+                label="Main gain is handled by the Mixxx built-in mixer">
+                <description>
+                    When enable, the master potentiometer on top right column of the mixer will drive the main gain of the Mixxx internal mixer as well as the hardware built-in mixer in the device. This is useful if you aren't relying on the hardware output, but will create conflict if enabled when using the hardware outputs.
```

Comment:
Not sure if the word `master` is problematic here - the device is physically labelled with `Master` so if it is, perhaps I can put `"` around it or the `:hwbtn:` marker?